addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -655,11 +655,36 @@ function factory(dependencies) {
                     if (!dataChannel || dataChannel.readyState !== 'open') {
                         continue;
                     }
-                    dataChannel.send(JSON.stringify({
-                        event,
-                        channelId: this.channel.id,
-                        payload,
-                    }));
+                    try {
+                        dataChannel.send(JSON.stringify({
+                            event,
+                            channelId: this.channel.id,
+                            payload,
+                        }));
+                    } catch (e) {
```

Comment:
Considering the following comment is present on the `dataChannel.onopen` that is causing our traceback, I suggest to move this try catch into that handler only, and leave the other calls to `_notifyPeers` alone.

```js
/**
 * FIXME? it appears that the track yielded by the peerConnection's 'ontrack' event is always enabled,
 * even when it is disabled on the sender-side.
 */
```